src/main/pages/che-7/installation-guide/con_che-configmaps-and-their-behavior.adoc

```diff
@@ -11,16 +11,55 @@ A link:https://docs.openshift.com/container-platform/latest/builds/setting-up-tr
 
 link:https://docs.openshift.com/container-platform/latest/applications/operators/olm-what-operators-are.html[Operators] are software extensions to Kubernetes that use link:https://docs.openshift.com/container-platform/latest/applications/crds/crd-managing-resources-from-crds.html[custom resources] to manage applications and their components.
 
-Che installed using the Operator provides the user with two automatically generated `configMaps`:
+Che installed using the Operator provides the user with an automatically generated `che` `configMap`.
 
-* `che` (main)
-* `custom` (additional)
+The `che` `configMap` contains the main properties for the installation and is in sync with the information stored in the CheCluster Custom Resource file. User modifications of the `che` `configMap` after installing Che using the Operator are automatically overwritten by values that the Operator obtains from the `CheCluster` Custom Resource. 
 
-* The `che` `configMap` contains the main properties for the installation and is in sync with the information stored in the CheCluster Custom Resource file. User modifications of the `che` `configMap` after installing Che using the Operator are automatically overwritten by values that the Operator obtains from the `CheCluster` Custom Resource. +
```

Comment:
I would replace:

*main properties __for the installation__*

by 

*main properties __for the Che server__*